docs/content/gameobject/world-commands.md

```diff
@@ -32,145 +32,78 @@ The commands documentation is:
 
 
 ## How to send and receive world commands
-We provide the following types for sending and receiving world commands:
+We provide the following type for sending and receiving world commands:
 
-  * `WorldCommands.Requirable.WorldCommandRequestSender`
-  * `WorldCommands.Requirable.WorldCommandResponseHandler`
+  * `WorldCommandSender`
 
-> We do not provide a `WorldCommands.Requirable.WorldCommandRequestHandler` as the command will be directly handled by the SpatialOS Runtime.
+> We do not provide a `WorldCommandReceiver` as the command will be directly handled by the SpatialOS Runtime.
 
-Both of these objects can be injected without any condition. A MonoBehaviour that requires only these will be enabled as soon as the associated GameObject is created.
+The `WorldCommandSender` can be injected without any condition. A MonoBehaviour that requires only these will be enabled as soon as the associated GameObject is created.
```

Comment:
```suggestion
The `WorldCommandSender` can be injected without any condition. A MonoBehaviour that requires only this will be enabled as soon as the associated GameObject is created.
```